### ISSUE TITLE
CodeSandbox examples broken, and workaround also broken

### ISSUE THREAD

**Issue by dantasse** [downshift-js/downshift#1598]
Basically reopening #1570. Thanks to @dlong500 for explaining this and describing how to work around it by disabling the "Sandbox beta editor" option. Unfortunately, that option no longer exists, so as far as I can tell, all the examples are inaccessible.

**Comment by yusijs** [comment]
There is a workaround for it - add `moduleview=1` to the url params

see https://github.com/codesandbox/codesandbox-client/discussions/8350

**Comment by dlong500** [comment]
Yes, the workaround was just released in the last couple weeks.  A small PR to the downshift examples index page can update things to work with the new parameter.  I'll look into a PR soon.

**Comment by silviuaavram** [comment]
@dlong500 can you help me understand what do we need to change in the sandbox repo in order to get things working? Thanks!

**Comment by silviuaavram** [comment]
Closing this as duplicate.

https://github.com/downshift-js/downshift/issues/1570

@dlong500 once we agree on a proper fix I will help to deliver it to the repo.

**Comment by dlong500** [comment]
@silviuaavram I think the only things that needs to get updated at this point is to add `?moduleview=1` to any links pointing to the codesandbox repository, and also to update the language on the index page of the codesandbox repo to get rid of the instructions about changing to "Current Module View".

Oh, and the self-referencing link on the index page of the repo should also include the `?moduleview=1` query parameter for completeness and clarity.  As long as that query parameter is present the examples work just fine now that codesandbox added that feature.

**Comment by silviuaavram** [comment]
Done. Thanks for figuring it out!

**Comment by dlong500** [comment]
@silviuaavram it looks like the codesandbox link in the docs is now redirected and it loses the `moduleview=1` query parameter.  Can someone update the link to point directly to the current codesandbox repo?

To be clear, the current link in the docs points to:
https://codesandbox.io/s/github/kentcdodds/downshift-examples?moduleview=1

but this then gets redirected to:
https://codesandbox.io/p/sandbox/github/kentcdodds/downshift-examples

For some reason, even adding the `moduleview` parameter to the new URL still doesn't reliably work, but if it is added along with some `file` parameters it appears to stick.

Can we get the examples link updated to point the below URL?

https://codesandbox.io/p/sandbox/github/kentcdodds/downshift-examples?file=%2Fsrc%2Findex.js&moduleview=1

**Comment by silviuaavram** [comment]
Sure, let me change these.
